charts/kube-prometheus-stack/templates/grafana/configmap-dashboards.yaml

```diff
@@ -13,7 +13,7 @@ items:
     namespace: {{ template "kube-prometheus-stack-grafana.namespace" $ }}
     labels:
       {{- if $.Values.grafana.sidecar.dashboards.label }}
-      {{ $.Values.grafana.sidecar.dashboards.label }}: {{ ternary $.Values.grafana.sidecar.dashboards.labelValue "1" (not (empty $.Values.grafana.sidecar.dashboards.labelValue)) | quote }}
+      {{ $.Values.grafana.sidecar.dashboards.label }}: "1"
```

Comment:
damn, the python script was not changed in: https://github.com/prometheus-community/helm-charts/pull/1782

i guess removing it now is kind of a breaking change. should we add the change now or handle this in a separate pr?

Reply:
nevermind. fixed it...